ReactSkia/sdk/RNSKeyCodeMapping.h

```diff
@@ -54,32 +54,32 @@ typedef enum rnsKeyAction{
     RNS_KEY_Release
 }rnsKeyAction;
 static std::string RNSKeyMap[RNS_KEY_End]={
-    "Right",
-    "Left",
-    "Up",
-    "Down",
-    "Select",
-    "PowerOff",
-    "AudioMedia",
-    "Menu",
-    "AudioLowerVolume",
-    "AudioRaiseVolume",
-    "ChannelUp",
-    "ChannelDown",
-    "Help",
-    "Refresh",
-    "AudioStop",
-    "AudioPlay",
-    "AudioPause",
-    "Back",
-    "AudioNext",
-    "AudioRewind",
-    "AudioForward",
-    "AudioPrev",
-    "Green",
-    "Red",
-    "Blue",
-    "Yellow",
+    "right",
+    "left",
+    "up",
+    "down",
+    "select",
+    "powerOff",
+    "audioMedia",
+    "menu",
+    "audioLowerVolume",
+    "audioRaiseVolume",
+    "channelUp",
+    "channelDown",
+    "help",
+    "refresh",
+    "audioStop",
```

Comment:
regarding these key names , we may have to revisit these strings similar to Android key names . Refer - https://github.com/react-native-tvos/react-native-tvos/commit/b3d81dfd8604263671b9e006f8f00786b0f8fcfb#